ReactWindows/ReactNative/Touch/TouchHandler.cs

```diff
@@ -97,7 +97,16 @@ private void OnPointerConcluded(TouchEventType touchEventType, PointerRoutedEven
                 UpdatePointerForEvent(pointer, e);
                 DispatchTouchEvent(touchEventType, _pointers, pointerIndex);
 
-                --_activePointers;
+                if (_activePointers == pointerIndex + 1)
+                {
```

Comment:
Thanks again and great find @ebragge! As we talked about, I think the first conditional check (which is to only reduce the count when the "last" pointer is released) is probably not needed, and we can just reset the IDs when all pointers are released.